src/test/java/cz/muni/fi/pa165/bluebat/entity/ReservationTest.java

```diff
@@ -0,0 +1,141 @@
+package cz.muni.fi.pa165.bluebat.entity;
+
+import cz.muni.fi.pa165.bluebat.PersistenceTravelAgencyApplicationContext;
+import org.junit.jupiter.api.Assertions;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.Test;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.PersistenceException;
+import javax.persistence.PersistenceUnit;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+/**
+ * @author Ondrej Vaca
+ */
+
+@ContextConfiguration(classes = PersistenceTravelAgencyApplicationContext.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+public class ReservationTest extends AbstractTestNGSpringContextTests {
```

Comment:
Ještě mi chybí pozitivní test, že jde vložit do DB plně inicializovaná rezervace.